### ISSUE TITLE
watch metrics & PlaybackSpeedPlugin

### ISSUE THREAD

**Issue by byflipper** [clappr/clappr-stats#35]
Is there any way of updating 'watch' metrics in line with the playback speed plugin. 

When a video is played at x2 speed for 10 seconds, clappr-stats still reports it as 10 seconds, but I expect it to report 20 seconds. Or when the video is played at 0.5 speed for 10 seconds, it should be logged as 5 seconds. Is this possible? 
Thanks.

**Comment by leandromoreira** [comment]
I don't know, it seems wrong to use player's speed multiplier as base to increase the watch, but it might be only my opinion, thoughts @lucasrodcosta @joaopaulovieira @jhonatangomes @towerz ?

Maybe we could create `real_time_watch` and `effective_time_watch`.

**Comment by towerz** [comment]
I agree with @leandromoreira, perhaps we should have separate time units for actual time spent watching and the video effective watched time.

**Comment by byflipper** [comment]
@leandromoreira & @towerz 
Thanks for your comments.  
Is it possible to do this? Is there any way that I can get support for this because I really need this feature. 
Thanks for any help.
